src/ram/StringConstant.h

```diff
@@ -0,0 +1,59 @@
+/*
+ * Souffle - A Datalog Compiler
+ * Copyright (c) 2013, 2014, Oracle and/or its affiliates. All rights reserved
+ * Licensed under the Universal Permissive License v 1.0 as shown at:
+ * - https://opensource.org/licenses/UPL
+ * - <souffle root>/licenses/SOUFFLE-UPL.txt
+ */
+
+/************************************************************************
+ *
+ * @file StringConstant.h
+ *
+ * Defines string constants
+ *
+ ************************************************************************/
+
+#pragma once
+
+#include "ram/Expression.h"
+#include "ram/Node.h"
+#include "souffle/RamTypes.h"
+#include "souffle/utility/StringUtil.h"
```

Comment:
May not be needed?

Reply:
I forgot to reply on this. The idea of this PR is to make strings explicit in RAM. Before we referred to strings by their index number in the symbol table. With the new StringConstant, we can refer to strings directly.